Makefile

```diff
@@ -115,20 +116,26 @@ run: go.build
 
 .PHONY: cobertura reviewable manifests submodules fallthrough test-integration run crds.clean
 
-service: 
-	@if [ "$(ServiceID)" = "" ]; then \
-		echo "Error: Please specify value of 'ServiceID'."; \
+# NOTE(muvaf): ACK Code Generator is a separate Go module, hence we need to
```

Comment:
@muvaf can we pin this in our `go.mod` and invoke it with `go generate` like we do for other tools? (i.e. `crossplane-tools`)

Reply:
The code generator needs non-Go files (templates) that exist in its repo to work. But when we run `go run` without cloning, those files do not come in. We get the following error:
```
Error: template crossplane/apis/doc.go.tpl not found in any base search path: Template Not Found
exit status 1
```

Seems like this can be achieved with the new [`go:embed`](https://tip.golang.org/pkg/embed/) directive, but that's not used in ACK yet.